build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+###########################################################################
+#
+# Copyright 2020 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+#
+###########################################################################
+# rtl8721csm_download.sh
+
+if [ -n "$2" ];then
+	PORT="$2"
+else
+	PORT="ttyUSB1"
+fi
+
+WARNING="\n Port $PORT is selected\n\n
+	############################################\n
+	WARNINGS:\n
+	1. Make sure the board is in DOWNLOAD MODE.\n
```

Comment:
Is there any way to check the board is DOWNLOAD MODE or not?

Reply:
Hi @jeongchanKim ,
There are two ways to check if AmebaD is in download mode.
1) You can open SSH clients (like Putty, Tera Term) to check. If AmebaD is in download mode, you will see the message. (Reminder to disconnect it after checking to release the occupation of the serial port.)
2) More obviously, if AmebaD is in download mode, one of the led will be bilking.

Thanks!